### ISSUE TITLE
Additional Problem status type:  'in-progress' needs MS code changes

### ISSUE THREAD

**Issue by FrankSylvia** [nsmenon/mathspring-V2#64]
Recently we incompletely implemented a third status type: in-progress.  This is the new default (starting) status for problems.  'Testable' will now mean that the problem authoring is complete and its ready for testing.  But we ran into problems. First, MS should show in-progress (and testable) problems to test-users and it does not. 

Second, "there are difficulties with previewing in-progress from the MsAdmin" (Danielle). 
Basically it means that problems that are "in-progress" should show up in the following screen (see attached). 

<img width="855" alt="Screen Shot 2019-09-19 at 9 13 59 AM" src="https://user-images.githubusercontent.com/46008380/65615245-08a32780-df87-11e9-9cfb-dc65d7ef44b9.png">

 Only "development" student users can see this. Create a new user (System testing (developer view)) to see this window and be able to debug.

**Comment by FrankSylvia** [comment]
Moved to new repo as issue #4

**Comment by FrankSylvia** [comment]
Moved to new repo as issue #4
